Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt

```diff
@@ -17,72 +17,74 @@
 package com.example.compose.jetchat
 
 import android.os.Bundle
-import android.view.Menu
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
-import androidx.core.view.GravityCompat
-import androidx.drawerlayout.widget.DrawerLayout
+import androidx.compose.material.rememberScaffoldState
+import androidx.compose.runtime.Providers
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.platform.setContent
+import androidx.compose.ui.viewinterop.AndroidViewBinding
+import androidx.core.os.bundleOf
 import androidx.navigation.findNavController
-import androidx.navigation.ui.AppBarConfiguration
-import androidx.navigation.ui.navigateUp
-import androidx.navigation.ui.setupWithNavController
-import com.google.android.material.navigation.NavigationView
+import com.example.compose.jetchat.components.JetchatScaffold
+import com.example.compose.jetchat.conversation.AmbientBackPressedDispatcher
+import com.example.compose.jetchat.conversation.backPressHandler
+import com.example.compose.jetchat.databinding.ContentMainBinding
 
 /**
- * Main activity for the app. Shows a drawer and a toolbar rendered with traditional Views, for now.
+ * Main activity for the app.
  */
 class NavActivity : AppCompatActivity() {
 
-    private lateinit var appBarConfiguration: AppBarConfiguration
-    private lateinit var drawerLayout: DrawerLayout
+    // Used for navigation events between fragments.
```

Comment:
This ViewModel only takes care of the drawer, right?

Reply:
Yeah but potentially can be used for more things so I wouldn't say it's just for the drawer.

Reply:
Maybe remove the comment? It's confusing after going to ViewModel and see nothing about navigation there

Reply:
Done